tools/ci/gha-install-libmesh.sh

```diff
@@ -0,0 +1,24 @@
+
+#!/bin/bash
+set -ex
+
+CURRENT_DIR=$(pwd)
+
+# libMESH install
+cd $HOME
```

Comment:
```suggestion
# libMESH install
pushd $HOME
```
See `popd` suggestion below

Reply:
Cool! I wasn't aware of this. Much cleaner.